Source/WebKit/UIProcess/Extensions/Cocoa/API/WebExtensionContextAPIScriptingCocoa.mm

```diff
@@ -141,6 +142,35 @@
     completionHandler(std::nullopt);
 }
 
+void WebExtensionContext::scriptingRegisterScripts(const Vector<WebExtensionRegisteredScriptParameters> &parameters, CompletionHandler<void(WebExtensionDynamicScripts::Error)>&& completionHandler)
```

Comment:
`const Vector<WebExtensionRegisteredScriptParameters>& ` instead and the same for the methods below

Reply:
const Vector<WebExtensionRegisteredScriptParameters> &scripts